tensorboard/webapp/webapp_data_source/tb_server_data_source.ts

```diff
@@ -14,29 +14,37 @@ limitations under the License.
 ==============================================================================*/
 import {Injectable} from '@angular/core';
 import {HttpClient} from '@angular/common/http';
-
 import {from} from 'rxjs';
 
 import {PluginsListing} from '../types/api';
 
+import {DataApiRouteConverter} from './data_api_route_converter';
+import {DataSource} from './data_source';
+
 /** @typehack */ import * as _typeHackRxjs from 'rxjs';
 
 @Injectable()
-export class TBServerDataSource {
+export class TBServerDataSource implements DataSource {
   // TODO(soergel): implements WebappDataSource
```

Comment:
Can we now drop this TODO, or explain what more we plan to address later?

Reply:
Honestly... idk what it means and I rather not touch the part of the code that is not related. Feel free to file an issue :)